### ISSUE TITLE
Magic prayer about colic in BLorient12053 (ጸሎት፡ በእንተ፡ ቁርጸት፡ ወውግዓት፡ ደስክ፡) - Deleting existing IDs concerning colic (doublets and redundant IDs)

### ISSUE THREAD

**Issue by CarstenHoffmannMarburg** [SChAth/ScAthiop#2746]
@eu-genia @MagdaKrzyz @karljonaskarlsson @nafisa-valieva 

I found a prayer in BLorient12053 with the following incipit: ጸሎት፡ በእንተ፡ ቁርጸት፡ ወውግዓት፡ ደስክ፡ 

There are a number of work records about prayer against colic with very similar incipits: 

LIT4604Salot - ጸሎት፡ በእንተ፡ ሕማመ፡ ቍርጸት፡

LIT4622Prayer - ጸሎት፡ በእንተ፡ ሕማመ፡ ቍርጠት፡ 

LIT4623Prayer - ጸሎት፡ በእንተ፡ ሕማመ፡ ቍርጠት፡ (being a doublet of CAe 4622 with the same attestation in BAVet77)

LIT4595Salot - ጸሎት፡ በእንተ፡ ሕማመ፡ ቍርጸት፡

I want to create a new record for the prayer in BLorient12053. I want to delete three of the four existing LIT IDs noted above. All of them have the same incipit (ቍርጠት being just a phonetic variant of ቍርጸት). Disambiguation is not possible at the moment, because all witnesses do not have available images. Can I proceed?

**Comment by eu-genia** [comment]
Is the new one surely different? Would it make sense to have a more generic Prayer against colic for all?

I leave it to your judgement, just make sure all manuscript records are updated, thank you!

**Comment by karljonaskarlsson** [comment]
If I see it correctly, LIT4622Prayer and LIT4623Prayer have longer incipits. I would keep them separate.

For LIT4595Salot and LIT4604Salot, the incipit is too short to say anything.

I think Zhenia's suggestion to create a generic record makes sense. The prayer in BLorient12053 plus LIT4595Salot plus LIT4604Salot could be given the generic ID, whereas LIT4622Prayer and LIT4623Prayer could remain separate and be connected to the generic record with a relation.

At least, that would be one way of doing it. But of course, feel free to make a different decision :)

**Comment by CarstenHoffmannMarburg** [comment]
> If I see it correctly, LIT4622Prayer and LIT4623Prayer have longer incipits. I would keep them separate.

Yes! Sorry, I was too quick or mistaken about it, when I opened this issue.

> I think Zhenia's suggestion to create a generic record makes sense. The prayer in BLorient12053 plus LIT4595Salot plus LIT4604Salot could be given the generic ID, whereas LIT4622Prayer and LIT4623Prayer could remain separate and be connected to the generic record with a relation.

Yes, transforming one of them into a generic ID is exactly what I wanted to do. However, I think, I would prefer to create a new ID for the prayer in BLorient12053. The mention of ወውግዓት፡ and ደስክ፡ are two features, that makes it more specific to dedicate a record to it.

**Comment by CarstenHoffmannMarburg** [comment]
I had a closer look to LIT4595Salot and I discovered, that it is described as "Prayer against colic in a mixture of Amharic and Gǝʿǝz." in the abstract. I think, the reference to Amharic is something, that makes this prayer possibly different from the others. For this reason, I changed my mind and want to keep all the existing IDs and suffice myself with adding some relations to the records.

I am sorry, that I have overlooked this detail before!

**Comment by karljonaskarlsson** [comment]
> I had a closer look to LIT4595Salot and I discovered, that it is described as "Prayer against colic in a mixture of Amharic and Gǝʿǝz." in the abstract. I think, the reference to Amharic is something, that makes this prayer possibly different from the others. For this reason, I changed my mind and want to keep all the existing IDs and suffice myself with adding some relations to the records.
> 
> I am sorry, that I have overlooked this detail before!

I also overlooked the mention of Amharic. I agree with your conclusion that adding relations will be good!
